test/WorkerExtensionTests/Cosmos/UtilitiesTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.Functions.Worker.Extensions.CosmosDB;
+using Xunit;
```

Comment:
nit - usings ordering

Reply:
Done! Following this standard to be clear:

SA1210: Using directives must be ordered alphabetically by namespace.
SA1208: System using directives must be placed before other using directives.